openmetadata-ui/src/main/resources/ui/src/components/Classifications/ClassificationDetails/ClassificationDetails.tsx

```diff
@@ -480,14 +480,16 @@ const ClassificationDetails = forwardRef(
         )}
 
         <div className="m-b-sm m-t-xs" data-testid="description-container">
-          <Description
+          <DescriptionV1
             className={classNames({
-              'opacity-60': isClassificationDisabled,
+              'opacity-60': !isClassificationDisabled,
```

Comment:
Hello @harsh-vador , I think here we should have 
```suggestion
              'opacity-60': isClassificationDisabled,
```